### ISSUE TITLE
Test parachain env: local rococo

### ISSUE THREAD

**Issue by vovacha** [playzero/zero-network#138]
- [ ] Generate Collator list with keys, generate Sudo key
- [ ] Generate test `chain_spec`
- [ ] Test with local rococo relaychain setup:
  - [ ] Block production
  - [ ] Check if all collators are working
  - [ ] Test RPC node with the load balancer

**Comment by vovacha** [comment]
This task is blocked by:

- #129 
- #121 

@2075 @5-mark

**Comment by vovacha** [comment]
Since we will use a third-party service to run infrastructure, this ticket is not relevant anymore.
Also, testing with real PROD keys doesn't make much sense locally. Closing this task as "won't do".
